developers/integrations/compute-infrastructure/modal/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Modal
+sidebar_position: 1
+---
+
+[Modal](https://modal.com/) provides a serverless platform that enables developers to easily deploy and automatically scale applications with on-demand access to GPUs and a custom high-performance container runtime.
```

Comment:
Consider breaking this into shorter sentences:

[Modal](https://modal.com/) provides a serverless platform that has on-demand access to GPUs and a custom high-performance container runtime. Use Modal to easily deploy and automatically scale high-performance applications.